stand/efi/loader/main.c

```diff
@@ -920,7 +921,7 @@ acpi_detect(void)
 		if ((rsdp = efi_get_table(&acpi)) == NULL)
 			return;
 
-	sprintf(buf, "0x%016llx", (unsigned long long)rsdp);
+	sprintf(buf, "0x%016"PRIxPTR, (uintptr_t)rsdp);
```

Comment:
We compile with -Werror, so casting a pointer to an integer of a different size should cause the build to fail.
iirc I only noticed this because the GCC build failed. Clang seemingly ignores -Wpointer-to-int-cast when the size of the integer is bigger than that of the pointer's, which is probably why it doesn't fail on 32-bit arm